java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java

```diff
@@ -19,15 +19,104 @@
 
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
-import org.openqa.selenium.StandardSeleniumTests;
-import org.openqa.selenium.devtools.DevToolsTests;
+import org.junit.runners.model.InitializationError;
 
-
-@RunWith(Suite.class)
-@Suite.SuiteClasses({
-    StandardSeleniumTests.class,
-    ChromeOptionsFunctionalTest.class,
-    DevToolsTests.class
-})
+@RunWith(ChromeDriverTests.ChromeTestRunner.class)
 public class ChromeDriverTests {
-}
+  /* Use a custom suite to enable use of Class.forName, which is not allowed
+      by the annotations. This isolates tests run by ChromeDriver from changes
+      in the build tools. */ 
```

Comment:
We're not planning on shifting build tools again for a while, so I'm not sure that this is a concern.